### DISCUSSION TITLE
v0.13.0 is out!

### DISCUSSION THREAD

**Discussion by dignifiedquire** [n0-computer/iroh#2126] [Announcements]
* Install! https://iroh.computer/docs/install
* Check the [release notes](https://github.com/n0-computer/iroh/releases/tag/v0.13.0) for Details

**Comment by pratikpc** [comment]
@dignifiedquire is there a reason it has not been marked as latest?

0.12.0 is still marked as the latest release on GitHub

Caused some problems for @xialvjun [here](https://github.com/ScoopInstaller/Main/pull/5669)

Scoop looks at the GitHub Release marked as latest.

**Reply by dignifiedquire** [reply]
should be fixed, no idea why it wasn’t marked as such -.-
